app/controllers/application_controller.rb

```diff
@@ -1,2 +1,13 @@
 class ApplicationController < ActionController::Base
+    protect_from_forgery with: :exception
+
+    before_action :configure_permitted_parameters, if: :devise_controller?
+
+    protected
+
+         def configure_permitted_parameters
+              devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password)}
+
+              devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :email, :password, :current_password)}
```

Comment:
Out of scope, you are only working on `sign_up`. Not `account_update`

Reply:
Fixed mr mano